autoload/neomake/makers/ft/javascript.vim

```diff
@@ -1,11 +1,29 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#javascript#EnabledMakers() abort
-    return ['jshint', 'jscs',
+    return ['jshint', 'jscs', 'tsc',
```

Comment:
Should the be enabled automatically?  (given that you need special config for it from what I understand)